src/midiEditorCommands.cpp

```diff
@@ -203,15 +203,17 @@ struct MidiEventListData {
 		auto setting = format("list_{}", index);
 		char eventData[255] = "\0";
 		if (MIDIEditor_GetSetting_str(editor, setting.c_str(), eventData, sizeof(eventData))) {
+			MediaItem_Take* take = MIDIEditor_GetTake (editor);
 			string key, val;
 			istringstream s(eventData);
+			double eventPosPpq = -1.0;
+			double lengthPpq = -1.0;
 			while(getline(getline(s, key, '='), val, ' ')) {
 				if (key == "pos") {
-					auto eventPosQn = stof(val);
-					data.position = TimeMap2_QNToTime(nullptr, eventPosQn);
+					eventPosPpq = stof(val) * 960.0; // fixme: only works with the default ppq of 960.
```

Comment:
We use 960.0 twice, so please make it a const. It doesn't have to be at module level, just in a common scope for both uses is fine.